charts/ratify/templates/deployment.yaml

```diff
@@ -138,11 +138,7 @@ spec:
             name: {{ include "ratify.fullname" . }}-configuration
         - name: tls
           secret:
-            {{- if and .Values.provider.tls.crt .Values.provider.tls.key }}
             secretName: {{ include "ratify.fullname" . }}-tls
-            {{- else }}
-            secretName: ratify-cert-secret
```

Comment:
why is the secret removed? Is this option not valid anymore? https://github.com/deislabs/ratify/blob/b86fd6224d033421fc1127e6609bb999866aab0f/CONTRIBUTING.md?plain=1#L139

Reply:
@mluker wonder if we can remove the usage of this secret. Otherwise we need to choose key/cert among `ratify-cert-secret`, Values and helm generated cert/key.

Reply:
I don't have any objections with removing this.